src/goblint.ml

```diff
@@ -56,6 +56,8 @@ let main () =
         else
           None
       in
+      if AutoTune.isActivated "specification" && get_string "ana.specification" <> "" then
+        AutoTune.focusOnMemSafetySpecification ();
```

Comment:
This is too late! As soon as we have a file, CIL has already run!

Reply:
Should be fixed in https://github.com/goblint/analyzer/pull/1201/commits/4a4b6abc6edd18d1487084131a4f3bc038fe526f